tests/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3
+FROM python:3.7.5
```

Comment:
I would drop the patch version here: using 3.7 should be good enough.

Reply:
Unluckily, it is not enough as the regression has been introduced in Python 3.7 as well.
Anyway, looks like minio python library version 5.0.6 includes the fix so depending on it should fix the issue without pinning Python 3.7.5. I'm giving it a try later today!